EngineeringUnits/BaseUnit.cs

```diff
@@ -120,43 +128,43 @@ public override bool Equals(Object obj)
         public static bool operator ==(BaseUnit left, BaseUnit right)
         {
             if (left.Unit != right.Unit)            
-                throw new InvalidOperationException($"Cant do '==' on two differnt units!");
+                throw new InvalidOperationException($"Cant do '==' on two different units!");
```

Comment:
If you're going to change all these lines, may as well add the apostrophe in "Can't."